src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.searchBar {
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  font-weight: 300;
+  color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big {
+  background-size: 19px;
+  background-position-x: 26px;
+  font-size: 16px;
+  text-indent: 62px;
+  height: 70px;
+}
+
+.small {
+  background-size: 11px;
+  background-position-x: 13px;
+  font-size: 14px;
+  text-indent: 33px;
+  height: 42px;
+}
+
+.searchBar::placeholder {
+  font-family: Avenir-Book, sans-serif;
+}
+
+.big::placeholder {
+  text-indent: 62px;
+}
+
+.small::placeholder {
+  text-indent: 33px;
+}
+
+.formField {
+  margin-top: 20px;
+  display: block;
+}
+
+.big:hover {
+  background-color: #ffff;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small:hover {
+  background-color: #ffff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  background-image: url(images/Search.svg),
+    linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, repeat;
+  box-sizing: border-box;
```

Comment:
you are using box-sizing in many places. you can actually set it for the `*` selector. but this is the only property that is okay for the `*` selector